README.md

```diff
@@ -77,6 +77,24 @@ You can always checkout remote branches by specifying the remote as well
 
     wit update-pkg <package>::<remote>/<branch>
 
+### Restore a previous workspace
+
+If you have a matching pair `wit-lock.json` and `wit-workspace.json` from another workspace, you can create
+a new clean workspace by providing those files to the `restore` sub-command.
+
+    $ wit restore <new-workspace-name> -l </path/to/wit-lock.json> -w </path/to/wit-workspace.json>
+
+Dependencies are only specified by the `wit-lock.json` file, no other dependency resolution is performed.
+If you are keeping your wit-lock.json and wit-workspace.json under revision control you can do an in-place
+restore
+
+    $ git clone http://example.com/my_workspaces
+    $ cd my_workspaces/project1
+    $ ls
+    wit-lock.json wit-workspace.json
+    $ wit restore
+
+
```

Comment:
Rather than having two optional but related arguments `-l` and `-w`, what if instead you just point to a directory that is assumed to contain both?
```
wit restore <new-workspace-name> [--from-workspace|-w] </path/to/workspace>
```
This seems more consistent with the second form requiring no arguments.

Reply:
I guess my feeling was that no one would use the `-w` and `-l` much. I can change it to the directory style you've suggested.

On a somewhat related note, I feel like they should just be 2 sections in the one file

Reply:
That `--from-workspace` change is made now